scripts/jenkins-yoctobuild-build.sh

```diff
@@ -668,10 +610,7 @@ build_and_test_client() {
             bitbake mender-image-full-cmdline-rofs || bitbake_result=$?
         fi
 
-        if [[ $bitbake_result -eq 0 ]]; then
-            "$(dirname "$0")"/github_pull_request_status "success" "$board_name integration:${INTEGRATION_REV} poky:${POKY_REV} build completed" "$BUILD_URL" "${board_name}_${INTEGRATION_REV}_${POKY_REV}_build"
-        else
-            "$(dirname "$0")"/github_pull_request_status "failure" "$board_name integration:${INTEGRATION_REV} poky:${POKY_REV} build failed" "$BUILD_URL" "${board_name}_${INTEGRATION_REV}_${POKY_REV}_build"
+        if [[ $bitbake_result -ne 0 ]]; then
             exit $bitbake_result
         fi
```

Comment:
This can be simplified further: Just remove this whole `exit` block and remove all `bitbake_result` assignments. `set -e` should take care of the rest.

Reply:
Nice. Thanks